### ISSUE TITLE
Replace func

### ISSUE THREAD

**Issue by fireworx** [2shady4u/godot-sqlite#133]
could you implement the Replace feature

would be nice to have

**Comment by 2shady4u** [comment]
Hi @fireworx!

You can already use the Replace feature in the current version of this plugin:
https://www.sqlitetutorial.net/sqlite-replace-statement/

Example:
```gdscript
db.query("REPLACE INTO positions (title, min_salary) VALUES('Full Stack Developer', 140000);")
```

**Comment by fireworx** [comment]
yes, i know, but i use the feature, that you can take a hole dictionary
-> for example update_rows(tablename,where,dict)

;-)
